software/winrm/src/main/java/org/apache/brooklyn/location/winrm/WinRmMachineLocation.java

```diff
@@ -72,8 +72,10 @@
             "address",
             "Address of the remote machine");
 
-    public static final ConfigKey<Integer> WINRM_PORT = WinRmTool.PROP_PORT;
-    
+    public static final ConfigKey<Integer> WINRM_PORT = WinRmTool.PROP_PORT.getConfigKey();
+    public static final ConfigKey<Boolean> USE_SECURE_WINRM = WinRmTool.USE_SECURE_WINRM;
```

Comment:
Minor, we want `HTTPS` transport here, so a `USE_HTTPS_WINRM` could be a better name.